docs/getting-started.md

```diff
@@ -0,0 +1,75 @@
+---
+---
+
+A Couchbase-Elasticsearch data-replication system consists of three principal components:
+
+- a **Couchbase Server-cluster** of one or more nodes
+- an **Elasticsearch cluster** of one or more nodes
+- the **Elasticsearch Transport Plug-in**, installed in the Elasticsearch environment.
+
+This section provides a step-by-step procedure, whereby the three components can be installed, configured, and run.
+
+## Pre-requisites
+
+- Ubuntu 14.04 or 16.04
+- Java
+
+## Installation
+
+1. Download the Elasticsearch 2.4.0 package for Ubuntu.
```

Comment:
Anyone who's using ES 2.x already has it installed. For new users, we should recommend the latest version of 5.x.

Perhaps we should also mention that ES 6.0 is not supported yet.